helios-services/src/main/java/com/spotify/helios/master/OldJobReaper.java

```diff
@@ -46,12 +49,14 @@
   private static final Clock SYSTEM_CLOCK = new SystemClock();
   private static final long INTERVAL = 1;
   private static final TimeUnit INTERVAL_TIME_UNIT = TimeUnit.DAYS;
+  private static final DateFormat DATE_FORMATTER =
+      new SimpleDateFormat("EEE, d MMM yyyy HH:mm:ss Z");
```

Comment:
Good point. I've created a new PR using the thread-safe jodatime here https://github.com/spotify/helios/pull/874